doc/examples/apply_ufunc_vectorize_1d.ipynb

```diff
@@ -0,0 +1,2762 @@
+    "        Core dimensions on the inputs to exclude from alignment and\n",
+    "        broadcasting entirely. Any input coordinates along these dimensions\n",
+    "        will be dropped. Each excluded dimension must also appear in\n",
+    "        ``input_core_dims`` for at least one argument. Only dimensions listed\n",
+    "        here are allowed to change size between input and output objects.\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2019-12-30T15:55:24.910186Z",
+     "start_time": "2019-12-30T15:55:24.860198Z"
+    }
+   },
+   "outputs": [
+    {
+     "ename": "ValueError",
+     "evalue": "each dimension in `exclude_dims` must also be a core dimension in the function signature",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mValueError\u001b[0m                                Traceback (most recent call last)",
+      "\u001b[0;32m<ipython-input-6-70bee5bf07ac>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      4\u001b[0m     \u001b[0mair\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mlat\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      5\u001b[0m     \u001b[0mnewlat\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 6\u001b[0;31m     \u001b[0mexclude_dims\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mset\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"lat\"\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m  \u001b[0;31m# dimensions allowed to change size. Must be set!\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      7\u001b[0m )\n",
+      "\u001b[0;32m~/miniconda3/envs/dcpy/lib/python3.7/site-packages/xarray/core/computation.py\u001b[0m in \u001b[0;36mapply_ufunc\u001b[0;34m(func, input_core_dims, output_core_dims, exclude_dims, vectorize, join, dataset_join, dataset_fill_value, keep_attrs, kwargs, dask, output_dtypes, output_sizes, *args)\u001b[0m\n\u001b[1;32m    982\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0mexclude_dims\u001b[0m \u001b[0;32mand\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0mexclude_dims\u001b[0m \u001b[0;34m<=\u001b[0m \u001b[0msignature\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mall_core_dims\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    983\u001b[0m         raise ValueError(\n\u001b[0;32m--> 984\u001b[0;31m             \u001b[0;34m\"each dimension in `exclude_dims` must also be a \"\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    985\u001b[0m             \u001b[0;34m\"core dimension in the function signature\"\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u00
```

Comment:
The first sentence looks like it was cut short and should have ended with something like "... provides some information on what is going on."

Also, nit: `print` is not a statement in py3 anymore so this should be "... but the output of our `print` call" or "... but our `print` call output"

Reply:
Fixed both. Thanks.

I've also stripped the output.